fluent-package/apt/systemd-test/install-newly.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -exu
+
+. $(dirname $0)/../commonvar.sh
+
+sudo apt install -V -y \
+    /vagrant/${distribution}/pool/${code_name}/${channel}/*/*/*_${architecture}.deb
+
+systemctl status fluentd
+
+sudo apt remove -y fluent-package
+
+test $(systemctl status fluent > /dev/null 2>&1; echo $?;) -eq 4
```

Comment:
It should be fluentd.

Reply:
Thanks!
Fixed: https://github.com/fluent/fluent-package-builder/compare/b0048368905c98229ff582b9fbeb9b61ac96a7ac..965fd9ab6b10df2a4cfc10db73382f74b5c470ee